adafruit_ble_apple_media.py

```diff
@@ -28,6 +33,8 @@
 # Disable protected access checks since our private classes are tightly coupled.
 # pylint: disable=protected-access
 
+AppleMediaServiceType = Union("AppleMediaService", Type["AppleMediaService"])
+
```

Comment:
So that this line of code is not evaluated by CircuitPython (where the `Union` type isn't available), it needs to be moved up into the `try ... except ImportError` block, something like
```
try:
    from typing import Union, Type
    AppleMediaServiceType = Union("AppleMediaService", Type["AppleMediaService"])
except ImportError:
    pass
```